packages/connectivity/package.json

```diff
@@ -44,9 +44,11 @@
     "http-proxy-agent": "^5.0.0",
     "https-proxy-agent": "^5.0.0",
     "jsonwebtoken": "^8.5.1",
+    "lodash": "^4.17.21",
```

Comment:
[req] I don't think we should include lodash now. Please also see my comment below, where this is used.